homeassistant/components/rest/switch.py

```diff
@@ -43,6 +44,7 @@
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
         vol.Required(CONF_RESOURCE): cv.url,
+        vol.Optional(CONF_STATE_RESOURCE): cv.url,
```

Comment:
Should we make it a template ?

Reply:
The concept doesn't really apply to any of the other REST platforms. Feels too specific to make it a template/constant.